src/operator/optimizer_op.cu

```diff
@@ -261,8 +261,7 @@ NNVM_REGISTER_OP(ftml_update)
 .set_attr<FCompute>("FCompute<gpu>", FTMLUpdate<gpu>);
 
 NNVM_REGISTER_OP(adam_update)
-.set_attr<FCompute>("FCompute<gpu>", AdamUpdate<gpu>)
-.set_attr<FComputeEx>("FComputeEx<gpu>", AdamUpdateEx<gpu>);
```

Comment:
@eric-haibin-lin Is it safe to remove `AdamUpdateEx`?

Reply:
Oops. I don't think we should remove it. Had removed it temporarily. Will undo it.

Thanks.